docs/conf.py

```diff
@@ -116,6 +120,80 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ["_static"]
 # html_css_files = ["custom.css"]
-
+# Add jupyter configuration
+# Add jupyter configuration
+jupyter_sphinx_thebelab_config = {
+    "theme": "dark",  # Change to "light" if you prefer light theme
+    "loader_color": "#069",  # This is the spinner color when cells are executing
+}
+
+# You can also add styling options for highlight.js which renders the code
+jupyter_sphinx_highlight_css = "default"  # Other options: "github", "monokai", etc.
 # Show in footer when the docs were last updated.
 html_last_updated_fmt = "%b %d, %Y"
+
+
+# Override jupyter-sphinx styling to match Furo theme
+def setup(app):
+    # Create CSS that inherits colors from Furo theme
+    app.add_css_file("jupyter-sphinx-override.css")
+    css_content = """
+    /* Override jupyter-sphinx styling to match Furo theme */
+    div.jupyter_container {
+        background-color: var(--color-background-secondary);
+        border: 1px solid var(--color-background-border);
+        box-shadow: none;
+        margin-bottom: 1em;
+    }
+    
+    .jupyter_container div.code_cell {
+        background-color: var(--color-code-background);
+        border: 1px solid var(--color-background-border);
+        border-radius: 0.2rem;
+    }
+    
+    .jupyter_container div.code_cell pre {
+        background-color: var(--color-code-background);
+        color: var(--color-code-foreground);
+        font-family: var(--font-stack--monospace);
+    }
+    
+    div.jupyter_container div.highlight {
+        background-color: var(--color-code-background);
+    }
+    
+    .jupyter_container .output {
+        background-color: var(--color-background-secondary);
+        padding: 0.5em;
+        border-top: 1px solid var(--color-background-border);
+    }
+    
+    .jupyter_container div.output pre {
+        background-color: var(--color-background-secondary);
+        color: var(--color-foreground-primary);
+        font-family: var(--font-stack--monospace);
+    }
+    
+    /* Fix for output highlighting */
+    .jupyter_container .output .highlight {
+        background-color: var(--color-background-secondary);
+    }
+    
+    /* Style for the prompt */
+    .jupyter_container .highlight .gp {
+        color: var(--color-brand-primary);
+        font-weight: bold;
+    }
+    
+    /* Style for code comments */
+    .jupyter_container .highlight .c1 {
+        color: var(--color-foreground-secondary);
+        font-style: italic;
+    }
+    """
+    static_dir = os.path.join(app.outdir, "_static")
+    os.makedirs(static_dir, exist_ok=True)
+
+    # Write the CSS file
```

Comment:
[nit]: Likely can remove as the line that write the file is self-evidently writing the CSS file. 

```suggestion
```